packages/lesswrong/server/scripts/utils.js

```diff
@@ -59,3 +59,9 @@ export const wrapVulcanAsyncScript = (name, scriptFunc) => async (...args) => {
     console.log(`============ ${name} exiting ============`)
   }
 }
+
+export function getFieldsWithAttribute(schema, attributeName) {
+  return _.filter(
+    _.map(schema, (fieldSchema, fieldName) => fieldSchema[attributeName] ? fieldName : null 
+  ), f => f != null)
```

Comment:
This works, but could be more elegant; I think if you have just an `_.filter`, it will work pretty much the same way.